packages/editor-ui/src/router.ts

```diff
@@ -55,8 +55,7 @@ function getTemplatesRedirect(store: Store<IRootState>) {
 
 const router = new Router({
 	mode: 'history',
-	// @ts-ignore
-	base: window.BASE_PATH ?? '/',
+	base: window.BASE_PATH === '/{{BASE_PATH}}/' ? '/' : window.BASE_PATH,
 	scrollBehavior(to, from, savedPosition) {
```

Comment:
this unfortunately breaks the routing when hosted under a subfolder.
let me quickly test this.

Reply:
@ivov I updated the check to detect dev mode instead.